Resources/Prototypes/_Nyano/Entities/Objects/Devices/pda.yml

```diff
@@ -44,15 +44,25 @@
 
 - type: entity
   parent: BasePDA
-  id: GuardPDA
-  name: guard PDA
+  id: PrisonGuardPDA
+  name: prison guard PDA
   description: Red to hide the stains of prisoner blood.
   components:
   - type: Pda
-    id: GuardIDCard
+    id: PrisonGuardIDCard
```

Comment:
Same for this line; the linter is failing on this because **Prison**GuardIDCard doesn't exist